extras/windows/pre.txt

```diff
@@ -0,0 +1,4 @@
+Welcome to the Hazelcast CLC Install Wizard.
```

Comment:
```suggestion
Welcome to the Hazelcast CLC Installation Wizard.
```

Reply:
It seems both the "Install Wizard" and "Installation Wizard" are used, as far as I've checked on the internet.